src/src/tx_main.cpp

```diff
@@ -608,45 +633,44 @@ void setup()
   POWERMGNT.init();
   Radio.currFreq = GetInitialFreq(); //set frequency first or an error will occur!!!
   Radio.Begin();
-  Radio.SetSyncWord(UID[3]);
+  //Radio.SetSyncWord(UID[3]);
   POWERMGNT.setDefaultPower();
 
-
-  SetRFLinkRate(RATE_200HZ);
+  SetRFLinkRate(RATE_DEFAULT); // fastest rate by default
   crsf.Begin();
-  //hwTimer.init(); //enable this for debug
+  hwTimer.init();
+  hwTimer.stop(); //comment to automatically start the RX timer and leave it running
 }
 
 void loop()
 {
+  while(UpdateParamReq){
+    HandleUpdateParameter();
+  }
 
 #ifdef FEATURE_OPENTX_SYNC
   // Serial.println(crsf.OpenTXsyncOffset);
 #endif
 
-HandleUpdateParameter();
-
-  //updateLEDs(isRXconnected, ExpressLRS_currAirRate_Modparams->TLMinterval);
-
   if (millis() > (RX_CONNECTION_LOST_TIMEOUT + LastTLMpacketRecvMillis))
   {
     isRXconnected = false;
-#if defined(TARGET_R9M_TX) || defined(TARGET_R9M_LITE_TX)
+    #if defined(TARGET_R9M_TX) || defined(TARGET_R9M_LITE_TX)
     digitalWrite(GPIO_PIN_LED_RED, LOW);
-#endif
+    #endif
   }
   else
   {
     isRXconnected = true;
-#if defined(TARGET_R9M_TX) || defined(TARGET_R9M_LITE_TX)
+    #if defined(TARGET_R9M_TX) || defined(TARGET_R9M_LITE_TX)
     digitalWrite(GPIO_PIN_LED_RED, HIGH);
-#endif
+    #endif
   }
 
-  float targetFrameRate = (ExpressLRS_currAirRate_Modparams->rate * (1.0 / TLMratioEnumToValue(ExpressLRS_currAirRate_Modparams->TLMinterval)));
-  PacketRateLastChecked = millis();
-  PacketRate = (float)packetCounteRX_TX / (float)(PACKET_RATE_INTERVAL);
-  linkQuality = int((((float)PacketRate / (float)targetFrameRate) * 100000.0));
+  // float targetFrameRate = (ExpressLRS_currAirRate_Modparams->rate * (1.0 / TLMratioEnumToValue(ExpressLRS_currAirRate_Modparams->TLMinterval)));
```

Comment:
Need this?

Reply:
Gotta fix the calculation for TQLY at some point, leave me the code for reference :)